pkg/domain/infra/abi/play.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/containers/podman/v4/pkg/util"
 	"github.com/containers/podman/v4/utils"
 	"github.com/coreos/go-systemd/v22/daemon"
-	"github.com/docker/distribution/reference"
+	"github.com/distribution/reference"
```

Comment:
Can we use `"github.com/containers/image/v5/docker/reference"` instead?  Looks like a `go-import` issue to me choosing the wrong package.

Reply:
sure, that seems better.

Amended and pushed a new version

Reply:
Thanks!